integration-tests/verify_metadata_headers.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
+
+import os
+import sys
+import linecache
+
+metadata_header_file = sys.argv[1]
+if not os.path.exists(metadata_header_file):
+    print "Error, specified file not found: " + metadata_header_file
+    sys.exit(1)
+
+DISPLAY_NAME_METADATA_HEADER = linecache.getline(metadata_header_file, 1)
+DESCRIPTION_METADATA_HEADER = linecache.getline(metadata_header_file, 2)
+DATATYPE_METADATA_HEADER = linecache.getline(metadata_header_file, 3)
+PRIORITY_METADATA_HEADER = linecache.getline(metadata_header_file, 4)
+NORMALIZED_COLUMN_HEADER = linecache.getline(metadata_header_file, 5)
+
+metadata_headers = [DISPLAY_NAME_METADATA_HEADER, DESCRIPTION_METADATA_HEADER, DATATYPE_METADATA_HEADER, PRIORITY_METADATA_HEADER]
+
+# boolean tracking first four lines are commented out (metadata) and fifth line is not (regular column headers)
+headers_present = all([True if header.startswith("#") and len(header.rstrip().split("\t")) == len(NORMALIZED_COLUMN_HEADER.rstrip().split("\t")) else False for header in metadata_headers]) and not NORMALIZED_COLUMN_HEADER.startswith("#")
+display_name_metadata_header_is_valid = all([True if display_name else False for display_name in DISPLAY_NAME_METADATA_HEADER.replace("#","").rstrip().split("\t")])
+description_metadata_header_is_valid = all([True if description else False for description in DESCRIPTION_METADATA_HEADER.replace("#","").rstrip().split("\t")])
+datatype_metadata_header_is_valid = all([True if datatype == "NUMBER" or datatype == "STRING" else False for datatype in DATATYPE_METADATA_HEADER.replace("#","").rstrip().split("\t")])
+priority_metadata_header_is_valid = all([True if priority.isdigit() else False for priority in PRIORITY_METADATA_HEADER.replace("#","").rstrip().split("\t")])
+
+if not all([headers_present, display_name_metadata_header_is_valid, description_metadata_header_is_valid, datatype_metadata_header_is_valid, priority_metadata_header_is_valid]):
+    print "Error, one or more header checks failed."
```

Comment:
Maybe loop through each header check (e.g. datatype_metadata_header_is_valid) and then output an error about which ones fail.  E.g. "Error, the following header checks failed: datatype_metadata_header_is_valid" and maybe even output the invalid header.